README.md

```diff
@@ -9,13 +9,23 @@ The AWS Toolkit for Visual Studio Code is an extension for working with AWS serv
 
 The Toolkit is available from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/itemdetails?itemName=AmazonWebServices.aws-toolkit-vscode).
 
-This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull requests or any positive contribution.
+This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull 
+requests or any positive contribution. See [CONTRIBUTING](CONTRIBUTING.md) for how to help.
 
 ## Documentation
 
 -   The [Quick Start Guide](extension-readme.md) provides a summary of the Toolkit and the most common tasks.
 -   The [User Guide](https://docs.aws.amazon.com/console/toolkit-for-vscode/welcome) contains detailed instructions for getting up and running with the Toolkit.
 
+## Feedback
```

Comment:
I'm thinking we may want to swap this and the other change (or at least slot this in over). I'd expect more people to participate by commenting rather than contributing.

Reply:
The other change is inline, since it didn't have enough content for its own section.